packages/app/components/Queue/TAQueueCard.tsx

```diff
@@ -23,6 +23,7 @@ export default function TAQueueCard({
   question,
   onOpen,
 }: TAQueueCardProps): ReactElement {
+  const waitTime = now.getMinutes() - question.createdAt.getMinutes();
```

Comment:
i think you forgot to copy over `now`

Reply:
```suggestion
  const now = new Date();
  const waitTime = now.getMinutes() - question.createdAt.getMinutes();
```

Reply:
i did and then committed but never pushed -- there's a new bug there you might wanna check out :P